README.md

```diff
@@ -0,0 +1,35 @@
+# GitHub Keeper
+
+GitHub keeper is a CLI tool that helps to unify our repositories.
+
+Features:
+
+* List all Exasol repos where you are admin (`list-my-repos`)
+* Create / Update the branch protection for the default branch according to our standards (`create-branch-protection`)
+
+#### Installation
+
+1. Install Go language. Minimum version is 1.13. 
+2. Install dependendent packages: 
+```
+cd $HOME/<PATH TO THIS REPO>/product-integration-tool-chest/github-keeper
+go get ./...
+```
+
+#### Usage
```

Comment:
```suggestion
## Usage
```

Reply:
Maybe good idea to follow first, second then third level headings.

Reply:
yes. Copy and paste issue...